exercises/static/exercises/drone_cat_mouse/web-template/launch/drone_cat_mouse.launch

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <arg name="mav_name" default="crazyflie2"/>
+  <!-- <arg name="world_name" default="basic"/> -->
+  <arg name="enable_internal_model_controller" default="true"/>
+  <arg name="enable_logging" default="false" />
+  <arg name="enable_ground_truth" default="true" />
+  <arg name="enable_state_estimator" default="false" />
+  <arg name="enable_mellinger_controller" default="false"/>
+  <arg name="headless" default="true"/>
+  <arg name="log_file" default="$(arg mav_name)" />
+  <arg name="paused" value="false"/>
+  <arg name="use_sim_time" value="true"/>
+  <arg name="debug" default="false"/>
+  
+  <arg name="gui" default="false"/>
+  <arg name="csvFilesStoring" default="false"/>
+  <arg name="csvFilesStoringTime" default="15.0"/> <!-- seconds -->
+  <arg name="user_account" default="giuseppe"/>
+  <!-- The following line causes gzmsg and gzerr messages to be printed to the console
+      (even when Gazebo is started through roslaunch) -->
+  <arg name="verbose" default="true"/>
+
+  <!-- The following lines simulate the world in Gazebo. The physic engine properties
+        are set up in the file "basic_crazyflie.world" file -->
+  <env name="GAZEBO_MODEL_PATH" value="${GAZEBO_MODEL_PATH}:$(find rotors_gazebo)/models"/>
```

Comment:
we should include this environment variables in RADI configuration (.env)

Reply:
commented out in the updates and .env was already updated in the reviewed changes